Src/GoogleApis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -99,5 +110,45 @@ public Initializer(string tokenUrl)
         }
 
         #endregion
+
+        /// <summary>
+        /// Return whether code is running on Google Compute Engine.
+        /// </summary>
+        public static async Task<bool> IsRunningOnComputeEngine()
+        {
+            try

```

Comment:
Every time I close the paranthesis, VS auto fixes this. Do you seriously feel that this is such a big deal - it was incorrect to begin with that is being fixed in this checkin. It is seriously more work to prolong the error than to fix it.